python/ray/services.py

```diff
@@ -1285,6 +1294,9 @@ def build_java_worker_command(
     command += "-Dray.home={} ".format(RAY_HOME)
     command += "-Dray.log-dir={} ".format(os.path.join(session_dir, "logs"))
 
+    command += ("-Dray.raylet.config.num_workers_per_process_java=" +
+                "RAY_WORKER_NUM_WORKERS ")
```

Comment:
Use a prefix like `JAVA_WORKER_OPTION_` or `JAVA_WORKER_PLACEHOLDER_`?

Otherwise it's hard to be read as a placeholder to other developers.(I read it for a while to understand)

Reply:
Updated both placeholders to `RAY_WORKER_PLACEHOLDER_DYNAMIC_OPTION_` and `RAY_WORKER_PLACEHOLDER_NUM_WORKERS`.